### PR TITLE
[Telemetry Operator] Set output plugin alias and rewrite tag emitter name

### PR DESCRIPTION
**Pull request by Teneroy**
**Description**

Changes proposed in this pull request:

- Added logpipeline name to the name of input
- Added logpipeline name to the name of output
- Updated samples for telemetry-operator
- Adapted tests

The affected plugin fields are to be used by the fluent bit metrics as labels, which in turn make the Fluent Bit Grafana dashboard more descriptive. 

**Related issue(s)**
#15315

### CODE REVIEW

File: resources/telemetry/charts/fluent-bit/values.yaml [hunk 1]
```diff
@@ -366,6 +366,7 @@ config:
   inputs: |
     [INPUT]
         Name tail
+        Alias tail-tele
```
Comment (chrkl) [inline resources/telemetry/charts/fluent-bit/values.yaml:369]: Make "tail" a suffix to have it consistent.

File: resources/telemetry/charts/fluent-bit/values.yaml [hunk 2]
```diff
@@ -374,12 +375,12 @@ config:
...
         DB /data/flb_kube.db
         storage.type  filesystem
-
+ 
     [INPUT]
         Name tail
         Path /null.log
         Tag null.*
-        Alias dummy
+        Alias tail-dummy
```
Comment (a-thaler) [inline resources/telemetry/charts/fluent-bit/values.yaml:383]: Shouldn't it be called "null-dummy" as well as it is part of the "null" pipeline somehow

File: components/telemetry-operator/internal/fluentbit/config/builder/output.go [hunk 3]
```diff
@@ -29,9 +29,20 @@ func createOutputSection(pipeline *telemetryv1alpha1.LogPipeline, defaults Pipel
 func generateCustomOutput(output *telemetryv1alpha1.Output, fsBufferLimit string, name string) string {
 	sb := NewOutputSectionBuilder()
 	customOutputParams := parseMultiline(output.Custom)
+	pipelineName := ""
```
Comment (chrkl) [inline components/telemetry-operator/internal/fluentbit/config/builder/output.go:32]: Use `var pipelineName string` (same below)

**Review by skhalash — CHANGES_REQUESTED** [review]

**Pull request merged**